modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/internal/upgrade/v1_1_0/UpgradeDDMStructure.java

```diff
@@ -172,28 +169,16 @@ protected void upgradeDDMStructureVersionDefinition() throws Exception {
 		}
 	}
 
-	private String _convertExpression(
-		Collection<DDMFormField> ddmFormFields, String visibilityExpression) {
-
+	private String _convertExpression(String visibilityExpression) {
 		List<String> parameterValues =
 			ExpressionParameterValueExtractor.extractParameterValues(
 				visibilityExpression);
 
 		for (String parameterValue : parameterValues) {
-			if (Validator.isNull(parameterValue)) {
-				continue;
-			}
-
-			Stream<DDMFormField> ddmFormFieldsStream = ddmFormFields.stream();
-
-			boolean hasParameterValue = ddmFormFieldsStream.anyMatch(
-				ddmFormField -> ddmFormField.getProperty(
-					"name"
-				).equals(
-					parameterValue
-				));
+			if (Validator.isNull(parameterValue) ||
+				Validator.isNumber(parameterValue) ||
+				StringUtil.startsWith(parameterValue, StringPool.QUOTE)) {
```

Comment:
@marcosapmf, is apostrophe character a valid way to start an expression or just quote character is allowed?

Reply:
@natocesarrego, quote is enough to valid a parameter because you cannot create a visibility expression with `'US'` instead `"US"` on 7.0.x.

Reply:
Great! Thanks for the clarification, @marcosapmf. 😄